src/components/PlanningTool.jsx

```diff
@@ -16,7 +16,6 @@ import Modal from './Modal'
 import EditItemModal from './EditItemModal'
 import Constants from '../constants/Constants'
 import Legend from "./Legend";
-import * as propTypes from "prop-types";
```

Comment:
We do not use PropTypes anymore?

Reply:
It was an unused import. 
There is 'import PropTypes from "prop-types"' at line 13.